action.yml

```diff
@@ -34,5 +34,5 @@ outputs:
     description: 'Build result metadata'
 
 runs:
-  using: 'node16'
+  using: 'node20'
```

Comment:
Same as https://github.com/docker/login-action/pull/590#discussion_r1315431040 I think we should have wait and also bump to v5 as this is a major change and node 20 has some issues on some runners: https://github.com/actions/checkout/commit/97a652b80035363df47baee5031ec8670b8878ac#commitcomment-125836360

Reply:
ugh, should I revert then?

Reply:
I think so yes but checkout v4 update looks good

Reply:
> I think so yes but checkout v4 update looks good

Agreed, the checkout change can be cherry picked. Would node18 as runner work as node16 would go eol soon anyways so majority users would migrate to node 18 the next supported LTS version?

Reply:
I'll do it, 1min

Reply:
done v4.6.0